etc/profile-a-l/cointop.profile

```diff
@@ -0,0 +1,50 @@
+# Firejail profile for cointop
```

Comment:
firecfg.config?

Reply:
Sorry - have trouble understanding what you mean by this.  The profile was mostly generated by `firejail --build`; but on version 0.9.66.  I made some tweaks afterwards based on the template in the repo.

Reply:
https://github.com/netblue30/firejail/blob/afed1f5aac9692382ac02f11ffd78ba5ee7d369c/etc/templates/profile.template#L17-L19